v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/BigtableChangeStreamsToGcsOptions.java

```diff
@@ -0,0 +1,187 @@
+    String getBigtableInstanceId();
+
+    void setBigtableInstanceId(String bigtableInstanceId);
+
+    @TemplateParameter.Text(
+            order = 3,
+            description = "Bigtable Table ID",
+            helpText = "The Bigtable table ID to read change streams from.")
+    @Validation.Required
+    String getBigtableTableId();
+
+    void setBigtableTableId(String bigtableTableId);
+
+    @TemplateParameter.Text(
+        order = 4,
+        description = "Bigtable App Profile ID",
+        helpText = "The Bigtable App Profile ID to read change streams from.")
+    @Validation.Required
+    String getBigtableAppProfileId();
+
+    void setBigtableAppProfileId(String bigtableAppProfileId);
+
+    @TemplateParameter.DateTime(
+        order = 5,
+        optional = true,
+        description = "The timestamp to read change streams from",
+        helpText =
+            "The starting DateTime, inclusive, to use for reading change streams "
+                + "(https://tools.ietf.org/html/rfc3339). For example, 2022-05-05T07:59:59Z. Defaults to the "
+                + "timestamp when the pipeline starts.")
+    @Default.String("")
+    String getStartTimestamp();
+
+    void setStartTimestamp(String startTimestamp);
+
+    @TemplateParameter.DateTime(
+        order = 6,
+        optional = true,
+        description = "The timestamp to read change streams to",
+        helpText =
+            "The ending DateTime, inclusive, to use for reading change streams "
+                + "(https://tools.ietf.org/html/rfc3339). Ex-2022-05-05T07:59:59Z. Defaults to an infinite "
+                + "time in the future.")
+    @Default.String("")
+    String getEndTimestamp();
+
+    void setEndTimestamp(String startTimestamp);
+
+    @TemplateParameter.Enum(
+        order = 7,
+        enumOptions = {"TEXT", "AVRO"},
+        optional = true,
+        description = "Output file format",
+        helpText =
+            "The format of the output Cloud Storage file. Allowed formats are TEXT, AVRO. Default is AVRO.")
+    @Default.Enum("AVRO")
+    FileFormat getOutputFileFormat();
+
+    void setOutputFileFormat(FileFormat outputFileFormat);
+
+    @TemplateParameter.Duration(
+        order = 8,
+        optional = true,
+        description = "Window duration",
+        helpText =
+            "The window duration/size in which data will be written to Cloud Storage. Allowed formats are: Ns (for "
+                + "seconds, example: 5s), Nm (for minutes, example: 12m), Nh (for hours, example: 2h).",
+        example = "5m")
+    @Default.String("5m")
+    String getWindowDuration();
+
+    void setWindowDuration(String windowDuration);
+
+    @TemplateParameter.Text(
+        order = 11,
+        optional = true,
+        description = "Bigtable Metadata Table Id",
+        helpText = "Table ID used for creating the metadata table.",
+        example = "__change_stream_md_table")
+    @Default.String("")
+    String getBigtableMetadataTableId();
```

Comment:
Let's agree what we call this:

getBigtableMetadataTableTableId or getBigtableMetadataTableId

Reply:
I also have this param:

```
  @TemplateParameter.Text(
      order = 6,
      optional = true,
      description = "Cloud Bigtable metadata instance ID",
      helpText = "The Cloud Bigtable instance to use for the change streams connector metadata table.")
  @Default.String("")
  String getBigtableMetadataInstanceId();

  void setBigtableMetadataInstanceId(String value);
```

Let's agree if you add it or I remove it :)

Reply:
1) Lets call it getBigtableMetadataTableTableId as it matches what the beam connector has. https://team.git.corp.google.com/cloud-bigtable-team/cdc-beam/+/refs/heads/staging/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java#1656

2) I'll add this field